README.md

```diff
@@ -132,19 +131,19 @@ $ DEBUG=1 ./build/teleport start -d
 ```
 
 Keep the server running in this mode, and make your UI changes in `/dist` directory.
-Refer to [the webapps README](https://github.com/gravitational/webapps/blob/master/README.md) for instructions on how to update the Web UI.
+For instructions about how to update the Web UI, read [the `webapps` README](https://github.com/gravitational/webapps/blob/master/README.md.) file.
 
 #### Updating Web UI assets
 
-After you commit a change to [the webapps
+After you commit a change to [the `webapps`
 repo](https://github.com/gravitational/webapps), you need to update the Web UI
 assets in the `webassets/` git submodule.
 
-Use `make update-webassets` to update the `webassets` repo and create a PR for
+Run `make update-webassets` to update the `webassets` repo and create a PR for
 `teleport` to update its git submodule.
 
 You will need to have the `gh` utility installed on your system for the script
-to work. You can download it from https://github.com/cli/cli/releases/latest
+to work. You can download it for free. For installation instructions, read the [GitHub CLI installation](https://github.com/cli/cli/releases/latest) documentation.
```

Comment:
```suggestion
to work. For installation instructions, read the [GitHub CLI installation](https://github.com/cli/cli/releases/latest) documentation.
```

Reply:
That makes sense, superfluous info for the intended audience.